packages/providers/src/sdk/CryptKeeperInjectedProvider.ts

```diff
@@ -1,372 +1,108 @@
-      method: RPCAction.GET_CONNECTED_IDENTITY_DATA,
+      method: RPCExternalAction.INJECTOR_CONNECT_IDENTITY,
+      payload: { urlOrigin: this.connectedOrigin },
     }) as Promise<ConnectedIdentityMetadata>;
   }
 
-  /**
-   * Connects to an existing identity for the specified host.
-   *
-   * @param {IConnectIdentityRequestArgs} host - The host for which to connect to an identity.
-   * @returns {Promise<void>} A Promise that resolves when the connection is complete.
-   */
-  async connectIdentity({ host }: IConnectIdentityRequestArgs): Promise<void> {
-    await this.post({
-      method: RPCAction.CONNECT_IDENTITY_REQUEST,
-      payload: {
-        host,
-      },
-    });
-  }
-
-  /**
-   * Handles incoming messages from the extension.
-   *
-   * @param {IInjectedMessageData} event - The message event.
-   * @returns {unknown} The result of handling the event.
-   */
-  eventResponser = (event: MessageEvent<IInjectedMessageData>): unknown => {
-    const { data } = event;
-
-    if (data.target === "injected-injectedscript") {
-      if (EVENTS.includes(data.nonce as EventName)) {
-        const [, res] = data.payload;
-        this.emit(data.nonce as EventName, res);
-        return;
-      }
-
-      if (!promises.has(data.nonce.toString())) {
-        return;
-      }
-
-      const [err, res] = data.payload;
-      const { reject, resolve } = promises.get(data.nonce.toString())!;
-
-      if (err) {
-        reject(new Error(err));
-        return;
-      }
-
-      resolve(res);
-
-      promises.delete(data.nonce.toString());
-    }
-  };
-
-  /**
-   * Retrieves the host permissions for the specified host.
-   *
-   * @param {string} host - The host for which to retrieve the permissions.
-   * @returns {Promise<unknown>} A Promise that resolves to the host permissions.
-   */
-  async getHostPermissions(host: string): Promise<unknown> {
-    return this.post({
-      method: RPCAction.GET_HOST_PERMISSIONS,
-      payload: host,
-    });
-  }
-
-  /**
-   * Sets the host permissions for the specified host.
-   *
-   * @param {string} host - The host for which to set the permissions.
-   * @param {IHostPermission} permissions - The host permissions to set.
-   * @returns {Promise<unknown>} A Promise that resolves to the result of setting the host permissions.
-   */
-  async setHostPermissions(host: string, permissions?: IHostPermission): Promise<unknown> {
+  async getConnectedIdentity(): Promise<ConnectedIdentityMetadata> {
     return this.post({
-      method: RPCAction.SET_HOST_PERMISSIONS,
-      payload: {
-        host,
-        ...permissions,
-      },
-    });
-  }
-
-  /**
-   * Creates an identity for the specified host.
-   *
-   * @param {ICreateIdentityRequestArgs} host - The host for which to create an identity.
-   * @returns {Promise<void>} A Promise that resolves when the identity creation is complete.
-   */
-  async createIdentity({ host }: ICreateIdentityRequestArgs): Promise<void> {
```

Comment:
`createIdentity` is checked with e2e and this feature can be useful for creating identities from the dapp without switching to extension.

Reply:
I will remove this check from the e2e.